docs/cristin-proxy-swagger.yaml

```diff
@@ -1980,6 +1985,52 @@ components:
+            description: 'Approval authority. Only certain combinations of authority and applicationCode are allowed. REK: Regional committees for medical and health research ethics (en), Regionale komitéer for medisinsk og helsefaglig forskningsetikk (REK) (nb), Regionale komitéer for medisinsk og helsefaglig forskningsetikk (REK) (nn), application_code: ETICHAPPR or BIOBANK; NARA: The Norwegian Animal Research Authority (en), Forsøksdyrutvalget (nb), Forsøksdyrutvalet (nn), application_code: TESTANIMAL; NDPA: The Norwegian Data Protection Authority (en), Datatilsynet (nb), Datatilsynet (nn), application_code: SENSINFO; NMA: Norwegian Medicines Agency (en), Statens legemiddelverk (nb), Statens legemiddelverk (nn), application_code: DRUGTRIAL; DIRHEALTH: Norwegian Directorate of Health (en), Helsedirektoratet (nb), Helsedirektoratet (nn), application_code: BIOTECHN or MEDEQUIP.'
+            enum:
+              - REK
+              - NARA
+              - NDFA
+              - NMA
+              - DIRHEALTH
+          status:
+            type: string
+            description: 'Status of the application sent to the approval authority. NOTAPPLIED: Not applied (en), Ikke søkt (nb), Ikkje søkt (nn); APPLIED: Applied (en), Søkt (nb), Søkt (nn); APPROVED: Approved (en), Godkjent (nb), Godkjent (nn); DECLINED: Declined (en), Avvist (nb) Avvist (nn); REJECTION: Rejection (en), Avslag (nb), Avslag (nn).'
+            enum:
+              - NOTAPPLIED
+              - APPLIED
+              - APPROVED
+              - DECLINED
+              - REJECTION
+          applicationCode:
+            type: string
+            description: 'These codes apply only for applications (søknader) to the approval authorities. Only certain combinations of application_code and approved_by are allowed, see approved_by above. ETICHAPPR: Application for ethical approval of a health research project (en), Søknad om etisk godkjenning, Helseforskningsprosjekt (nb), Søknad om etisk godkjenning av helseforskingsprosjekt (nn); BIOBANK: Application for establishing a biobank (en), Søknad om etablering av biobank (nb), Søknad om etablering av biobank (nn); DRUGTRIAL: Application for a clinical drug trial (en), Søknad om legemiddelutprøving (nb), Søknad om legemiddelutprøving (nn); BIOTECHN: Application for medical use of biotechnology (en), Søknad om medisinsk bruk av bioteknologi (nb), Søknad om medisinsk bruk av bioteknologi (nn); MEDEQUIP: Application for clinical testing for medical equipment (en), Søknad om klinisk utprøving av medisinsk utstyr (nb), Søknad om klinisk utprøving av medisinsk utstyr (nn); TESTANIMAL: Application for testing on animals (en), Søknad om å utføre forsøk på dyr (nb), Søknad om å utføre forsøk på dyr (nn); SENSINFO: Application for license tp process sensitive personal information (en), Søknad om konsesjon for behandling av sensitive personopplysninger (nb), Søknad om konsesjon for handsaming av sensitive personopplysingar(nn).'
```

Comment:
The text here indicates that we will likely allow construction of invalid objects

Reply:
Can you elaborate?